README.rst

```diff
@@ -88,7 +89,10 @@ Before running tests, please ensure that following environment variables are set
     ==> BASIC_AUTH_PASSWORD
     ==> USER_LOGIN_EMAIL
     ==> USER_LOGIN_PASSWORD
-
+    ==> COURSE_RUN
+    ==> COURSE_DISPLAY_NAME
+    ==> COURSE_NUMBER
+    ==> COURSE_ORG
 
 To run all the tests:
 
```

Comment:
On second thought... @raeeschachar, maybe the paver e2e_test target should exclude the white label tests by default, and only run them with an include flag? Either way, I think we should try to be a little more clear about what this paver target does, because it doesn't exactly run all of the tests.